client/src/app/core/repositories/topics/topic-repository.service.ts

```diff
@@ -52,11 +52,21 @@ export class TopicRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCon
     }
 
     public getTitle = (titleInformation: TopicTitleInformation) => {
-        if (titleInformation.agenda_item_number()) {
-            return titleInformation.agenda_item_number() + ' · ' + titleInformation.title;
-        } else {
-            return titleInformation.title;
+        if (titleInformation.agenda_item_number) {
+            /**
+             * TODO: bug: for the list of speakers, agenda_item_number is property and not a function.
+             */
+            const itemNumber =
+                typeof titleInformation.agenda_item_number === 'function'
+                    ? titleInformation.agenda_item_number()
+                    : titleInformation.agenda_item_number;
+
+            if (itemNumber) {
+                return `${itemNumber} · ${titleInformation.title}`;
+            }
```

Comment:
@FinnStutzenstein that's more like it I suppose.
What confuses me is, that in some cases `agenda_item_number` is a property (list of speakers) and in others it's a function that returns a string

Reply:
Done!